code/src/pages/PopularList.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import '../css/popularList.css'
+
+const PopularList = ({movielist}) => {
+    return (
+        <div className="movie-container">
+            {movielist.map((movie) => (
+                <Link
+                   className="link-container"
```

Comment:
we couldn't see that you targeted this className, but instead targeted .movie-container a in the CSS, so maybe it could be removed.